rest/route/project_vars.go

```diff
@@ -0,0 +1,103 @@
+package route
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/util"
+
+	"github.com/pkg/errors"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+)
+
+////////////////////////////////////////////////////////////////////////
+//
+// GET /rest/v2/projects/{project_id}/variables
+
+type getProjectVarsHandler struct {
+	projectID string
+	sc        data.Connector
+}
+
+func makeGetVariablesByProject(sc data.Connector) gimlet.RouteHandler {
+	return &getProjectVarsHandler{
+		sc: sc,
+	}
+}
+
+func (h *getProjectVarsHandler) Factory() gimlet.RouteHandler {
+	return &projectIDGetHandler{
+		sc: h.sc,
+	}
+}
+
+func (h *getProjectVarsHandler) Parse(ctx context.Context, r *http.Request) error {
+	h.projectID = gimlet.GetVars(r)["project_id"]
+	return nil
+}
+
+func (h *getProjectVarsHandler) Run(ctx context.Context) gimlet.Responder {
+	projectVars, err := h.sc.FindProjectVarsById(h.projectID)
+	if err != nil {
+		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "problem getting variables for project '%s'", h.projectID))
+	}
```

Comment:
should we handle not found seperatly from database errors?

Reply:
This is how we handle finding projects as well, so I personally think it's fine.